src/io/flutter/inspector/InspectorService.java

```diff
@@ -36,8 +57,127 @@
  * inspector code running in the IDE.
  */
 public class InspectorService implements Disposable {
+
+  public static class Location {
+
+    public Location(VirtualFile file, int line, int column, int offset) {
+      this.file = file;
```

Comment:
Can `file` be made non-nullable? (What would it mean to have a null one?)

Reply:
done.